src/components/ProductCard/productcard.js

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+
+import CartLogo from "@/public/icons/minicart.svg";
+import { Button } from '../shared/Button';
+import { Link } from 'next/link';
+
+import Image from 'next/image'; // Импорт компонент Image из next/image
+import styles from './ProductCard.module.scss'; // Обратите внимание на .module.scss
+
+const ProductCard = ({ item, productUrl }) => {
+  const [quantity, setQuantity] = useState(1);
+  const [isDescriptionOpen, setDescriptionOpen] = useState(false);
+  const [isLinkCopied, setIsLinkCopied] = useState(false);
+
+  const incrementQuantity = () => {
+    if (item.ml > 0 && quantity < item.ml) {
+      setQuantity(quantity + 1);
+    }
+  };
+
+  const decrementQuantity = () => setQuantity(Math.max(1, quantity - 1));
+  const toggleDescription = () => setDescriptionOpen(!isDescriptionOpen);
+
+  const addToCart = () => {
+    if (item.ml > 0) {
+      alert(`Добавлено в корзину: ${quantity}`);
+    }
+  };
+
+  const stockText = (item.ml > 0) ? <span style={{ color: 'green' }}>В наличии</span> : <span style={{ color: 'red' }}>Нет в наличии</span>;
```

Comment:
```suggestion
  <span className={`${!!item.ml ? 'green' : 'red'}`}>
      {!!item.ml ? 'В наличии" :  'Нет в наличии'}
  </span>
```
И лучше вот так. Будет более читаемо.